### PR TITLE
Clean up profiles and add preview profile

### PR DESCRIPTION
**Pull request by howardjohn**
Preview is empty for now (that is, the same as default)

### CODE REVIEW

**Comment by PiotrSikora** [comment]
Looks good, thanks! We should probably fill it with experimental features, shouldn't we?

**Comment by howardjohn** [comment]
We should... the only one in particular we (in some chat) came up with was WASM and I don't think that is configurable through the charts yet? Its certainly not very useful if it is going to be empty though

File: operator/data/profiles/preview.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# The preview profile contains features that are experimental.
+# This is intended to explore new features coming to Istio.
+# Stability, security, and performance are not guaranteed - use at your own risk.
+apiVersion: operator.istio.io/v1alpha1
+kind: IstioOperator
+spec: {}
```
Comment (sdake) [inline operator/data/profiles/preview.yaml]: @howardjohn Are there other promoted defaults for 1.5 that should be reverted (and added here instead)?

Comment (howardjohn) [reply operator/data/profiles/preview.yaml]: The only ones I know of are WASM which is apparently not (yet) ready to be added here yet. So for now this is empty I suppose

File: operator/data/profiles/preview.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# The preview profile contains features that are experimental.
+# This is intended to explore new features coming to Istio.
+# Stability, security, and performance are not guaranteed - use at your own risk.
```
Comment (sdake) [inline operator/data/profiles/preview.yaml]: Might consider starting this yaml with this warning rather than concluding the comment with it.

**Review by sdake — APPROVED** [review]

**Review by elfinhe — APPROVED** [review]

**Pull request merged**